### PR TITLE
Fixing Year Field

### PR DESCRIPTION
**Pull request by yaroluchko**
As far as I can tell, this fixed the issue. Changed the year to be a Positive Integer Field rather than just an Integer Field.  Shows the year for me once I add a new document. Let me know if there is anything that I should have kept.

### CODE REVIEW

File: frontend/components/Documents.js [hunk 1]
```diff
@@ -138,8 +138,7 @@ const Documents = () => {
                                 <label htmlFor="year" className="col-2 col-form-label">Year</label>
                                 <div className="col">
                                     <input type="number" className="form-control"
-                                        id="year" value={newDocData.year}
-                                        max="9999"
+                                        id="year" value={newDocData.year} max = "9999"
```
Comment (MBJean) [inline frontend/components/Documents.js]: Let's remove the spaces around this equal sign here.

File: frontend/components/Documents.js [hunk 2]
```diff
@@ -68,7 +68,7 @@ const Documents = () => {
             },
             body: JSON.stringify({
                 title: newDocData.title,
-                year: typeof newDocData.year === "string" ? null : newDocData.year,
+                year: newDocData.year,
```
Comment (MBJean) [inline frontend/components/Documents.js:72]: As we discussed in person, this is likely the only change needed to resolve the immediate issue. If you can, I'd recommend undoing the other changes in this PR and we can simply merge this one change in. Let me know if you have any questions about how to do that!

**Review by MBJean — CHANGES_REQUESTED** [review]
Nice job troubleshooting this. See my commend above for next steps.

File: frontend/components/Documents.js (hunk 2 above)
Comment (yaroluchko) [reply frontend/components/Documents.js:72]: I just tried to undo the changes manually and hopefully that worked. Would there be an easier way to do that?

Comment (MBJean) [reply frontend/components/Documents.js:72]: You could use [`git-revert`](https://git-scm.com/docs/git-revert), but that seems unnecessarily complicated for this kind of change, so I think the manual adjustment is good.

**Review by MBJean — APPROVED** [review]
LGTM! Thanks for improving the codebase. When you're ready, you'll need to merge in the `main` branch locally, resolve the conflict that will occur in `frontend/components/Documents.js`, then push up your merge-and-conflict-resolve commit up here. With that done, you'll be able to click `Merge pull request` on the `Conversation` tab at the bottom of the page and announce your code change in the Slack room. [This part](https://urop.dhmit.xyz/git-workflow/) of the tech tutorial documentation outlines git workflows and includes a section about conflict resolution, and you can ask your team for assistance as well. Great work!

**Pull request merged**